src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -364,6 +364,7 @@
     <Compile Include="$(BclSourcesRoot)\System\WeakReference.cs" />
     <Compile Include="$(BclSourcesRoot)\System\WeakReferenceOfT.cs" />
     <Compile Include="$(BclSourcesRoot)\System\CLRConfig.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\StartupHookProvider.cs" />
```

Comment:
Nit, alpha order is good because it tends to reduce merge conflicts

Reply:
Fixed here and in master: https://github.com/dotnet/coreclr/pull/20028